src/Components/Home.js

```diff
@@ -25,11 +30,26 @@ function HomePage() {
 
   const posts = (
     <div className="posts">
-      <InstagramEmbed className="instaPost slide-up" url="https://www.instagram.com/p/C2-o7roPbfN" />
-      <InstagramEmbed className="instaPost slide-up" url="https://www.instagram.com/p/C23PyAWPYdW" />
-      <InstagramEmbed className="instaPost slide-up" url="https://www.instagram.com/p/CyRondZPVVS" />
-      <InstagramEmbed className="instaPost slide-up" url="https://www.instagram.com/p/Cxum20yOlkQ" />
-      <InstagramEmbed className="instaPost slide-up" url="https://www.instagram.com/p/CxrxWcQLpSb" />
+      <InstagramEmbed
+        className="instaPost slide-up"
```

Comment:
Probably shouldn't have these post uris hard-coded here, especially since we are hard-coding them elsewhere. Try to factor this out somehow?